components/Dialogs/src/MDCAlertController.h

```diff
@@ -167,10 +217,17 @@ typedef void (^MDCActionHandler)(MDCAlertAction *_Nonnull action);
 /**
  Title of the button shown on the alert dialog.
 
- Alert actions titles must be set in the actionWithTitle:handler: method.
+ Alert actions titles must be set using either of the actionWithTitle:[emphais:]handler: methods.
```

Comment:
Consider deleting this line and the similar line in the `emphasis` docs - using an init method to initialize properties is assumed knowledge of working with Objective-C APIs.

Reply:
Done.